docs/installation/online-installation.md

```diff
@@ -2,7 +2,7 @@
 title: Online Installation from Helm (Recommended)
 ---
 
-The best practice to deploy HAMi is using helm.
```

Comment:
This method may be best currently, but will not in future. We can use a safer text.